File: algorithms/Array/go/35-SearchInsertPosition/simple_search.go
func searchInsert(nums []int, target int) int {
    if len(nums) == 0 {
        return 0
    }
    var res = 0
    for idx := 0; idx<len(nums); idx++{
        if nums[idx] == target {
            return idx
        }
        if nums[idx] < target{
            res ++
        }
        if nums[idx] > target{
            return res
        }
    }
    return res
}

File: algorithms/Array/py/35-SearchInsertPosition/simple_search.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        """
            一般遍历查找
        """
        res = 0
        for idx, num in enumerate(nums):
            if num == target:
                return idx
            elif num < target:
                res = idx + 1
            else:
                return res
        return res
